### DISCUSSION TITLE
Animation font doesn't match Lottie preview

### DISCUSSION THREAD

**Discussion by roimulia2** [airbnb/lottie-ios#1692] [General]
## This issue is a:
- [V] Non-Crashing Bug (Visual or otherwise)

## Which Version of Lottie are you using?

Lottie 3.3.0

## What Platform are you on? 
- [V] iOS

## What Language are you in?
- [V] Swift

## Expected Behavior
Link to original Lottie:
https://lottiefiles.com/68713-secuence-motion-text

<img width="778" alt="Screen Shot 2022-04-16 at 17 32 08" src="https://user-images.githubusercontent.com/76737673/163679035-8761c5c9-9a5c-4d2e-b0f7-1f921273689a.png">

Image from Xcode:
<img width="329" alt="Screen Shot 2022-04-16 at 17 33 15" src="https://user-images.githubusercontent.com/76737673/163679068-1693f7da-fd83-41d4-96b5-55162efb948d.png">

## Code Example
I used the example code from the Lottie example project. Nothing special. Any reason for font mismatch?

**Comment by calda** [comment]
It looks like this animation has embedded font data for a font called `Druk-WideSuper`, which isn't supported on iOS. To use this animation on iOS, you'd need to set up an `AnimationFontProvider` to vend this font.
